components/function-controller/config/rbac/rbac_role_binding.yaml

```diff
@@ -9,4 +9,5 @@ roleRef:
   name: manager-role
 subjects:
 - kind: ServiceAccount
-  name: ""
+  name: default
+  namespace: system
```

Comment:
where is this namespace being created? via tekton?

Reply:
This file is generated and meant to be applied by kustomize, which replaces these entries, by the actual namespace.